base/server/src/main/java/com/netscape/cmscore/connector/HttpConnection.java

```diff
@@ -92,7 +92,9 @@ public HttpConnection(RemoteAuthority dest, ISocketFactory factory,
                 mHttpreq.setHeader("Content-Type", contentType );
             }
 
-            mHttpreq.setHeader("Connection", "Keep-Alive");
```

Comment:
I suppose this is no longer needed for HTTP/1.1?

Reply:
For HTTP1.1 the connection is keep alive unless `Connection: close` is provided. The opposite of 1.0.  [See the ref](https://www.rfc-editor.org/rfc/rfc2068#section-8.1.2).

Reply:
@fmarco76 So wouldn't that mean there might be places where you need to explicitly set connection: close now?  Where are they?

Reply:
HTTP1.1 default behaviour is to keep open the connection for multiple requests. Currently, the only communication I have modified is the CA <-> KRA which already was requesting to keep the connection open because the CA uses a pool of already opened connections to communicate. Therefore, it is the same as before but without the need of an explicit `Keep-Alive` (which is not supported by all servers and does not work well with proxies and in other cases).
When/if  we will move to HTTP1.1  other uses of the `HttpConnection`,  in other sub-systems, if there is not  connection pool and the connection is closed after the request then we should explicitly close (the server close in any case after a time-out, I think tomcat default is 30sec,  but for better resource usage it is better close)

Reply:
sounds good. thanks!